packages/slice-machine/rollup-server.config.js

```diff
@@ -19,5 +30,25 @@ export default {
       },
       tsconfig: "./server/tsconfig.json",
     }),
+    // rollup seems to ignore `undefined` plugins
+    SENTRY_AUTH_TOKEN &&
+      sentryRollupPlugin({
+        configFile: path.resolve(process.cwd(), "sentry-express.properties"),
+
+        // Auth tokens can be obtained from https://sentry.io/settings/account/api/auth-tokens/
+        // and need `project:releases` and `org:read` scopes
+        authToken: SENTRY_AUTH_TOKEN,
+
+        // Optionally uncomment the line below to override automatic release name detection
+        release: RELEASE_NUMBER,
+
+        // Specify the directory containing build artifacts
+        include: "./build/server",
```

Comment:
note: in the server folder there should only be an `index.js` file..... if there's not it means some other part of the build system is importing from server :)